### ISSUE TITLE
Incompatible with semantic-release v18

### ISSUE THREAD

**Issue by ismay** [semantic-release/changelog#168]
In updating to semantic release v18, I noticed that semantic-release/changelog specifies a peerDependency that doesn't allow semantic-release v18: https://github.com/semantic-release/changelog/blob/master/package.json#L64. Is this intentional, or a typo (as in a missing `=`)?

Currently it's blocking my upgrade to semantic-release v18, see the npm installation error here: https://github.com/ismay/stylelint-no-unsupported-browser-features/pull/191/checks?check_run_id=3639274728#step:4:13

**Comment by travi** [comment]
Correct. We still have several plugins that still need to be upgraded. For the time being, you can continue to use v17 until all of the plugins that you are using have been updated. 

If you would like to help, this is an example of the changes needed for each plugin. PRs are welcome.

**Comment by ismay** [comment]
> Correct. We still have several plugins that still need to be upgraded. For the time being, you can continue to use v17 until all of the plugins that you are using have been updated.

Ok cool. Yeah I was not sure whether it was an omission or intentional. Thanks for letting me know, I'll stick with v17 for now. Don't have time for contributing atm., just moved, but I will be cheering from the sidelines 😅.

**Comment by jdbruijn** [comment]
Wouldn't #167 with an update to the `engines` just be sufficient to be able to use this with v18, and perhaps just create a separate issue for the comment? I'm not a maintainer but 'd be happy to create a separate PR for the v18 update and updating engines to `>=12`, just let me know!

> lets add ls-engines to CI to alert us about this in the future

**Comment by travi** [comment]
> this is [an example](https://github.com/semantic-release/github/pull/410/files)

apologies that i didnt include the link above that i mentioned originally. i've updated my comment to reference the change that we made to one of the other plugins that has already been updated. the update to handle this properly isnt overly complex, we just need to get it handled. if you'd like to help us move this plugin forward more quickly than would happen otherwise, a PR would be welcomed.
